File: code/Server/src/main/java/com/MetaTreasures/MetaTreasures/web/dto/UserDto.java
package com.MetaTreasures.MetaTreasures.web.dto;

import com.MetaTreasures.MetaTreasures.core.model.User;
import lombok.*;

import java.time.Instant;
import java.util.List;

@Getter
@Setter
@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserDto {
    private Long userId;
    private String firebaseUid;
    private String email;
    private Instant createdAt;
    private Boolean verified;
    private List<BalanceDto> balances;
}

File: code/AndroidUI/app/src/main/java/com/metatreasures/metatreasures/data/dto/UserDto.kt
package com.metatreasures.metatreasures.data.dto

data class UserDto(
    val userId: Long,
    val firebaseUid: String,
    val email: String?,
    val createdAt: String,
    val verified: Boolean,
    val balances: List<BalanceDto>
)